src/data/bool.lean

```diff
@@ -132,6 +132,9 @@ theorem band_elim_right : ∀ {a b : bool}, a && b → b := dec_trivial
 
 @[simp] theorem bnot_true : bnot tt = ff := rfl
 
+@[simp] theorem bnot_ne (b : bool) : b ≠ !b :=
```

Comment:
I think a better lemma might be
```suggestion
@[simp] theorem ne_bnot (a b : bool) : a ≠ !b ↔ a = b :=
```
And the matching `bnot_ne : !a ≠ b ↔ a = b`

Reply:
Done.